.cargo/config.toml

```diff
@@ -1,2 +1,2 @@
-[target.x86_64-unknown-linux-gnu]
-rustflags = ["-C", "linker=clang", "-C", "link-arg=-fuse-ld=lld"]
\ No newline at end of file
+#[target.x86_64-unknown-linux-gnu]
+#rustflags = ["-C", "linker=clang", "-C", "link-arg=-fuse-ld=lld"]
```

Comment:
Please don't commit this. Feel free to keep this locally (or just install LLD), but it greatly helps incremental test times as it spends most of the time in the linker.

Reply:
I didn't have LLD installed so I temporarily commented it out. Didn't mean to commit it :sweat_smile: